lib/msal-common/src/request/CommonSilentFlowRequest.ts

```diff
@@ -22,4 +22,5 @@ export type CommonSilentFlowRequest = BaseAuthRequest & {
     account: AccountInfo;
     forceRefresh: boolean;
     tokenQueryParameters?: StringDict;
+    silentTokenRetrievalStrategy?: String;
```

Comment:
```suggestion
    silentTokenRetrievalStrategy?: SilentTokenRetrievalStrategy;
```

Again, make sure you import the enum into this file as well if you make this change.